src/app/home/home.component.ts

```diff
@@ -26,6 +26,8 @@ export class HomeComponent implements OnInit, OnDestroy {
   private subs = new Subscription();
 
   displayedColumns: string[] = ['name', 'race', 'class'];
+  columnsToDisplayWithExpand = [...this.displayedColumns, 'delete'];
```

Comment:
not needed but naming could be changed to be similar to displayedColumns, maybe displayedExpandedColumns ?

Reply:
replaced with displayedActionColunms because the additional columns will be edit and delete buttons.